dnf-testing.sh

```diff
@@ -153,3 +163,15 @@ run()
     exit $failed
 }
 [ "$action" = "run" ] && run
+
+shell()
+{
+    if [ -z "$devel" ];then
+        printf "\nsudo docker run -i --rm "$IMAGE" bash\n"
```

Comment:
Such prints are present also in the original code, that's why I have added it too.